Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0] mod_wsgi (pid=13854): Failed to exec Python script file '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0] mod_wsgi (pid=13854): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]   File "/home/a0130638/domains/webress.ru/public_html/site.wsgi", line 6, in <module> 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]     from myapp import app as application 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 28, in <module> 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]     file_handler = RotatingFileHandler('app.log', maxBytes=1024 * 1024 * 10, backupCount=5) 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/handlers.py", line 148, in __init__ 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]     BaseRotatingHandler.__init__(self, filename, mode, encoding, delay) 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/handlers.py", line 55, in __init__ 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]     logging.FileHandler.__init__(self, filename, mode, encoding, delay) 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/__init__.py", line 1143, in __init__ 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]     StreamHandler.__init__(self, self._open()) 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/__init__.py", line 1172, in _open 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0]     return open(self.baseFilename, self.mode, encoding=self.encoding) 127.0.0.1:61472
Jun 18 12:23:41 eir.from.sh httpd[13854]: [B] [66.249.77.165] webress.ru GET /robots.txt HTTP/1.0[13854]: 13854 [wsgi:error] [pid 13854] [client 66.249.77.165:0] PermissionError: [Errno 13] Permission denied: '/app.log' 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0] mod_wsgi (pid=25650): Failed to exec Python script file '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0] mod_wsgi (pid=25650): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]   File "/home/a0130638/domains/webress.ru/public_html/site.wsgi", line 6, in <module> 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]     from myapp import app as application 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 28, in <module> 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]     file_handler = RotatingFileHandler('app.log', maxBytes=1024 * 1024 * 10, backupCount=5) 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/handlers.py", line 148, in __init__ 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]     BaseRotatingHandler.__init__(self, filename, mode, encoding, delay) 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/handlers.py", line 55, in __init__ 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]     logging.FileHandler.__init__(self, filename, mode, encoding, delay) 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/__init__.py", line 1143, in __init__ 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]     StreamHandler.__init__(self, self._open()) 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/__init__.py", line 1172, in _open 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0]     return open(self.baseFilename, self.mode, encoding=self.encoding) 127.0.0.1:61472
Jun 18 13:53:41 eir.from.sh httpd[25650]: [B] [66.249.77.166] webress.ru GET /robots.txt HTTP/1.0[25650]: 25650 [wsgi:error] [pid 25650] [client 66.249.77.166:0] PermissionError: [Errno 13] Permission denied: '/app.log' 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0] mod_wsgi (pid=13399): Failed to exec Python script file '/home/a0130638/domains/webress.ru/public_html/site.wsgi'., referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0] mod_wsgi (pid=13399): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'., referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0] Traceback (most recent call last):, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]   File "/home/a0130638/domains/webress.ru/public_html/site.wsgi", line 6, in <module>, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]     from myapp import app as application, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 28, in <module>, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]     file_handler = RotatingFileHandler('app.log', maxBytes=1024 * 1024 * 10, backupCount=5), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/handlers.py", line 148, in __init__, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]     BaseRotatingHandler.__init__(self, filename, mode, encoding, delay), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/handlers.py", line 55, in __init__, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]     logging.FileHandler.__init__(self, filename, mode, encoding, delay), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/__init__.py", line 1143, in __init__, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]     StreamHandler.__init__(self, self._open()), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/logging/__init__.py", line 1172, in _open, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0]     return open(self.baseFilename, self.mode, encoding=self.encoding), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:41:37 eir.from.sh httpd[13399]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13399]: 13399 [wsgi:error] [pid 13399] [client 5.164.0.37:0] PermissionError: [Errno 13] Permission denied: '/app.log', referer: http://chatty.guru/ 127.0.0.1:61472
Jun 18 20:48:22 eir.from.sh httpd[26798]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[26798]: 26798 [wsgi:error] [pid 26798] [client 5.164.0.37:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 20:48:22 eir.from.sh httpd[26798]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[26798]: 26798 [wsgi:error] [pid 26798] [client 5.164.0.37:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 20:58:38 eir.from.sh httpd[13476]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13476]: 13476 [wsgi:error] [pid 13476] [client 5.164.0.37:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 20:58:38 eir.from.sh httpd[13476]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13476]: 13476 [wsgi:error] [pid 13476] [client 5.164.0.37:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 20:59:05 eir.from.sh httpd[13508]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13508]: 13508 [wsgi:error] [pid 13508] [client 5.164.0.37:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 20:59:05 eir.from.sh httpd[13508]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[13508]: 13508 [wsgi:error] [pid 13508] [client 5.164.0.37:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:02:16 eir.from.sh httpd[21230]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[21230]: 21230 [wsgi:error] [pid 21230] [client 5.164.0.37:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:02:16 eir.from.sh httpd[21230]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[21230]: 21230 [wsgi:error] [pid 21230] [client 5.164.0.37:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:02:34 eir.from.sh httpd[21262]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[21262]: 21262 [wsgi:error] [pid 21262] [client 5.164.0.37:0] The cache for model files in Transformers v4.22.0 has been updated. Migrating your old cache. This is a one-time only operation. You can interrupt this and resume the migration later on by calling `transformers.utils.move_cache()`., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:02:34 eir.from.sh httpd[21262]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[21262]: 21262 [wsgi:error] [pid 21262] [client 5.164.0.37:0] \r0it [00:00, ?it/s], referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:02:34 eir.from.sh httpd[21262]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[21262]: 21262 [wsgi:error] [pid 21262] [client 5.164.0.37:0] \r0it [00:00, ?it/s], referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:02:34 eir.from.sh httpd[21262]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[21262]: 21262 [wsgi:error] [pid 21262] [client 5.164.0.37:0] , referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:02:34 eir.from.sh httpd[21262]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[21262]: 21262 [wsgi:error] [pid 21262] [client 5.164.0.37:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:02:34 eir.from.sh httpd[21262]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[21262]: 21262 [wsgi:error] [pid 21262] [client 5.164.0.37:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:34:00 eir.from.sh httpd[12699]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[12699]: 12699 [wsgi:error] [pid 12699] [client 5.164.0.37:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:34:00 eir.from.sh httpd[12699]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[12699]: 12699 [wsgi:error] [pid 12699] [client 5.164.0.37:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:36:57 eir.from.sh httpd[19255]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[19255]: 19255 [wsgi:error] [pid 19255] [client 5.164.0.37:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 18 21:36:58 eir.from.sh httpd[19255]: [B] [5.164.0.37] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[19255]: 19255 [wsgi:error] [pid 19255] [client 5.164.0.37:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:24 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:24 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0] mod_wsgi (pid=32039): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'., referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0] Traceback (most recent call last):, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return self.wsgi_app(environ, start_response), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     response = self.handle_exception(e), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return cors_after_request(app.make_response(f(*args, **kwargs))), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return cors_after_request(app.make_response(f(*args, **kwargs))), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     response = self.full_dispatch_request(), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     rv = self.handle_user_exception(e), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return cors_after_request(app.make_response(f(*args, **kwargs))), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return cors_after_request(app.make_response(f(*args, **kwargs))), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     rv = self.dispatch_request(), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 83, in setIndex, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     prompt_helper = PromptHelper(max_input_size, num_output, max_chunk_overlap), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/llama_index/indices/prompt_helper.py", line 51, in __init__, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     self._tokenizer = tokenizer or globals_helper.tokenizer, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/llama_index/utils.py", line 61, in tokenizer, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     tokenizer = transformers.GPT2TokenizerFast.from_pretrained("gpt2"), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/transformers/tokenization_utils_base.py", line 1748, in from_pretrained, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     resolved_config_file = cached_file(, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/transformers/utils/hub.py", line 417, in cached_file, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     resolved_file = hf_hub_download(, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/huggingface_hub/utils/_validators.py", line 118, in _inner_fn, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return fn(*args, **kwargs), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/huggingface_hub/file_download.py", line 1159, in hf_hub_download, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     os.makedirs(storage_folder, exist_ok=True), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/os.py", line 213, in makedirs, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     makedirs(head, exist_ok=exist_ok), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/os.py", line 213, in makedirs, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     makedirs(head, exist_ok=exist_ok), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/os.py", line 213, in makedirs, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     makedirs(head, exist_ok=exist_ok), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/os.py", line 223, in makedirs, referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     mkdir(name, mode), referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:56:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0] PermissionError: [Errno 13] Permission denied: '/home/a0130638/.cache', referer: http://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0] mod_wsgi (pid=32039): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0] Traceback (most recent call last):, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return self.wsgi_app(environ, start_response), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     response = self.handle_exception(e), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return cors_after_request(app.make_response(f(*args, **kwargs))), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return cors_after_request(app.make_response(f(*args, **kwargs))), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     response = self.full_dispatch_request(), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     rv = self.handle_user_exception(e), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return cors_after_request(app.make_response(f(*args, **kwargs))), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return cors_after_request(app.make_response(f(*args, **kwargs))), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     rv = self.dispatch_request(), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 83, in setIndex, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     prompt_helper = PromptHelper(max_input_size, num_output, max_chunk_overlap), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/llama_index/indices/prompt_helper.py", line 51, in __init__, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     self._tokenizer = tokenizer or globals_helper.tokenizer, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/llama_index/utils.py", line 61, in tokenizer, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     tokenizer = transformers.GPT2TokenizerFast.from_pretrained("gpt2"), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/transformers/tokenization_utils_base.py", line 1748, in from_pretrained, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     resolved_config_file = cached_file(, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/transformers/utils/hub.py", line 417, in cached_file, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     resolved_file = hf_hub_download(, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/huggingface_hub/utils/_validators.py", line 118, in _inner_fn, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     return fn(*args, **kwargs), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/huggingface_hub/file_download.py", line 1159, in hf_hub_download, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     os.makedirs(storage_folder, exist_ok=True), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/os.py", line 213, in makedirs, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     makedirs(head, exist_ok=exist_ok), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/os.py", line 213, in makedirs, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     makedirs(head, exist_ok=exist_ok), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/os.py", line 213, in makedirs, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     makedirs(head, exist_ok=exist_ok), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]   File "/opt/rh/rh-python38/root/usr/lib64/python3.8/os.py", line 223, in makedirs, referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0]     mkdir(name, mode), referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:57:25 eir.from.sh httpd[32039]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[32039]: 32039 [wsgi:error] [pid 32039] [client 5.164.0.54:0] PermissionError: [Errno 13] Permission denied: '/home/a0130638/.cache', referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:59:20 eir.from.sh httpd[6134]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[6134]: 6134 [wsgi:error] [pid 6134] [client 5.164.0.54:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 20:59:21 eir.from.sh httpd[6134]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[6134]: 6134 [wsgi:error] [pid 6134] [client 5.164.0.54:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 21:00:02 eir.from.sh httpd[6162]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[6162]: 6162 [wsgi:error] [pid 6162] [client 5.164.0.54:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 21:00:02 eir.from.sh httpd[6162]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[6162]: 6162 [wsgi:error] [pid 6162] [client 5.164.0.54:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 21:02:16 eir.from.sh httpd[12101]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[12101]: 12101 [wsgi:error] [pid 12101] [client 5.164.0.54:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 21:02:16 eir.from.sh httpd[12101]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[12101]: 12101 [wsgi:error] [pid 12101] [client 5.164.0.54:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 21:02:33 eir.from.sh httpd[12133]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[12133]: 12133 [wsgi:error] [pid 12133] [client 5.164.0.54:0] There was a problem when trying to write in your cache folder (/home/a0130638/.cache/huggingface/hub). You should set the environment variable TRANSFORMERS_CACHE to a writable directory., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 19 21:02:33 eir.from.sh httpd[12133]: [B] [5.164.0.54] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb89b8e6bc157eb8e5 HTTP/1.0[12133]: 12133 [wsgi:error] [pid 12133] [client 5.164.0.54:0] None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used., referer: https://chatty.guru/ 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] mod_wsgi (pid=6108): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__ 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return self.wsgi_app(environ, start_response) 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     response = self.handle_exception(e) 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     response = self.full_dispatch_request() 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     rv = self.handle_user_exception(e) 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     rv = self.dispatch_request() 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args) 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 64, in setIndex 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     file = open("/home/a0130638/.cache/file.txt", "w") 127.0.0.1:61472
Jun 20 11:42:02 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] PermissionError: [Errno 13] Permission denied: '/home/a0130638/.cache/file.txt' 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0] mod_wsgi (pid=6146): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__ 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return self.wsgi_app(environ, start_response) 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     response = self.handle_exception(e) 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     response = self.full_dispatch_request() 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     rv = self.handle_user_exception(e) 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     rv = self.dispatch_request() 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args) 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 64, in setIndex 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     file = open("/home/a0130638/.cache/file.txt", "w") 127.0.0.1:61472
Jun 20 11:42:42 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0] PermissionError: [Errno 13] Permission denied: '/home/a0130638/.cache/file.txt' 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] mod_wsgi (pid=6108): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__ 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return self.wsgi_app(environ, start_response) 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     response = self.handle_exception(e) 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     response = self.full_dispatch_request() 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     rv = self.handle_user_exception(e) 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     rv = self.dispatch_request() 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args) 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 64, in setIndex 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     file = open("/home/a0130638/.cache/file.txt", "w") 127.0.0.1:61472
Jun 20 11:43:07 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] PermissionError: [Errno 13] Permission denied: '/home/a0130638/.cache/file.txt' 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0] mod_wsgi (pid=6146): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__ 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return self.wsgi_app(environ, start_response) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     response = self.handle_exception(e) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     response = self.full_dispatch_request() 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     rv = self.handle_user_exception(e) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     rv = self.dispatch_request() 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 64, in setIndex 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0]     file = open("/home/a0130638/.cache/file.txt", "w") 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6146]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6146]: 6146 [wsgi:error] [pid 6146] [client 5.164.8.0:0] PermissionError: [Errno 13] Permission denied: '/home/a0130638/.cache/file.txt' 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] mod_wsgi (pid=6108): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__ 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return self.wsgi_app(environ, start_response) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     response = self.handle_exception(e) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     response = self.full_dispatch_request() 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     rv = self.handle_user_exception(e) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     rv = self.dispatch_request() 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args) 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 64, in setIndex 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0]     file = open("/home/a0130638/.cache/file.txt", "w") 127.0.0.1:61472
Jun 20 11:43:09 eir.from.sh httpd[6108]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[6108]: 6108 [wsgi:error] [pid 6108] [client 5.164.8.0:0] PermissionError: [Errno 13] Permission denied: '/home/a0130638/.cache/file.txt' 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0] mod_wsgi (pid=23471): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__ 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     return self.wsgi_app(environ, start_response) 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     response = self.handle_exception(e) 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     response = self.full_dispatch_request() 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     rv = self.handle_user_exception(e) 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     rv = self.dispatch_request() 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args) 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 70, in setIndex 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0]     storageProject = f'{storage}{project["id"]}' 127.0.0.1:61472
Jun 20 11:54:01 eir.from.sh httpd[23471]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[23471]: 23471 [wsgi:error] [pid 23471] [client 5.164.8.0:0] TypeError: 'NoneType' object is not subscriptable 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0] mod_wsgi (pid=29161): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2213, in __call__ 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     return self.wsgi_app(environ, start_response) 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2193, in wsgi_app 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     response = self.handle_exception(e) 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     response = self.full_dispatch_request() 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     rv = self.handle_user_exception(e) 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     return cors_after_request(app.make_response(f(*args, **kwargs))) 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     rv = self.dispatch_request() 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/python/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args) 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 71, in setIndex 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0]     storageProject = f'{storage}{project["id"]}' 127.0.0.1:61472
Jun 20 11:57:11 eir.from.sh httpd[29161]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29161]: 29161 [wsgi:error] [pid 29161] [client 5.164.8.0:0] TypeError: 'NoneType' object is not subscriptable 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0] mod_wsgi (pid=29202): Failed to exec Python script file '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0] mod_wsgi (pid=29202): Exception occurred processing WSGI script '/home/a0130638/domains/webress.ru/public_html/site.wsgi'. 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0] Traceback (most recent call last): 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/site.wsgi", line 10, in <module> 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0]     from myapp import app as application 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0]   File "/home/a0130638/domains/webress.ru/public_html/myapp.py", line 191 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0]     return str(e) 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0]     ^ 127.0.0.1:61472
Jun 20 11:57:59 eir.from.sh httpd[29202]: [B] [5.164.8.0] webress.ru GET /index/5b45495fe70eb1cde1ec10f59ff2aa42df02c16b0506f4cb8we9b8e6bc157eb8e5 HTTP/1.0[29202]: 29202 [wsgi:error] [pid 29202] [client 5.164.8.0:0] SyntaxError: 'return' outside function 127.0.0.1:61472
